include/dd/Package.hpp

```diff
@@ -2153,6 +2178,63 @@ namespace dd {
             return r;
         }
 
+        std::map<std::string, double> getProbVectorFromDensityMatrix(dEdge e, double measurementThreshold) {
+            std::map<std::string, double> measuredResult = {};
+            double                        p0, p1;
+            double                        globalProbability;
+            double                        statesToMeasure;
+
+            dEdge::alignDensityEdge(&e);
+
+            statesToMeasure = std::pow(2, nqubits);
```

Comment:
why isn't `statesToMeasure` not directly initialised inline?

Reply:
Changed that.